src/agent/v8debugapi.js

```diff
@@ -59,7 +59,7 @@ var formatInterval = function(msg, interval) {
 
 var singleton;
 module.exports.create = function(logger_, config_, jsFiles_, sourcemapper_) {
-  if (singleton) {
+  if (singleton && !config_.force_) {
```

Comment:
Could we change the name to be more explicit even if this is for testing only? Maybe `forceNewAgent_`?